### PR TITLE
Create data-explorer-iot-system-properties.md

### PR DESCRIPTION
**Pull request by vladikbr**
Need different documentation for IoT Hub system properties

### CODE REVIEW

**Comment by PRMerger10** [comment]
@vladikbr : Thanks for your contribution! The author(s) have been notified to review your proposed change. @vladikb

**Comment by vladikbr** [comment]
@orspod, please review

**Comment by vladikbr** [comment]
#sign-off

**Comment by PRMerger7** [comment]
@vladikbr: I'm sorry - only the author of this article, @vladikb, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by orspod** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@orspod: I'm sorry - only the author of this article, @vladikb, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: data-explorer/includes/data-explorer-iot-system-properties.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+author: vladikb
```
Comment (orspod) [inline data-explorer/includes/data-explorer-iot-system-properties.md:2]:
```suggestion
author: orspod
```

**Comment by PRMerger10** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
#sign-off

**Comment by ktoliver** [comment]
New content generally should be submitted or moved to the private repo, but this include file looks good/basic in GitHub preview. 

However, blocking quality criteria listed in the internal contributor guide says that "Each new include file is used in at least one Markdown article. New include files can't be added to the repo before they are used in an article." 

@orspod We'll need to either close the PR or the pull request will need to be expanded to include a reference of the new include file in an MD file. 

Thanks!

**Comment by orspod** [comment]
@ktoliver - the article is in PR #708 (https://github.com/MicrosoftDocs/dataexplorer-docs/pull/708). I thought I could sign off on this first and then the other one. Will sign off on other one as well. Hope that is OK.
Agreed this content should've been submitted in the private repo. Gave that feedback to @vladikbr 
Please help me get this live.

**Comment by ktoliver** [comment]
Thanks for the info @orspod. Done.

**Pull request merged**